src/lib.rs

```diff
@@ -345,7 +345,6 @@ pub mod tests_cfg;
 mod util;
 
 pub use database::*;
-pub use driver::*;
```

Comment:
Instead of removing this line, I think we should add one `#[allow(clippy::)]`, because the content of the re-export depends on feature flags

Reply:
Fixed 
[138b9dc](https://github.com/SeaQL/sea-orm/pull/2056/commits/138b9dc75a006e5fcf2809455ff3aea534662d03)

I will add `#[allow(clippy::unused_xxx)]` later because I could not see clippy error in my local somehow...

Reply:
I have added `#[allow(unused_imports)]`
[20dc881](https://github.com/SeaQL/sea-orm/pull/2056/commits/20dc881d81bc7951c6efbfe4b8ba863f41b3f8a0)